.github/workflows/daily-workflow-ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Daily Workflow CI
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'database/scripts/lib/*'
+      - 'database/scripts/generate_report.rb'
+      - 'database/scripts/format_report.rb'
```

Comment:
Should it be here buildfarmer.db too? for the cases when there are modified columns and such

Reply:
Agreed

Reply:
Added in: b8dfd93